### PR TITLE
Add mime.types to prevent potential classpath conflicts

### PR DESCRIPTION
**Pull request by m3brown**
Travis tests began failing due to 'json' and 'csv' filetypes not being supported.  The issue was traced to the mime.types file in https://github.com/cndreisbach/ring.middleware.mime-extensions not being present in the classpath.

It appears that some lein environments load the mime.types from another qu dependency first:

```clojure
user=> (clojure.java.io/resource "mime.types")
#<URL jar:file:/home/vagrant/.m2/repository/com/amazonaws/aws-java-sdk/1.3.18/aws-java-sdk-1.3.18.jar!/mime.types>
```

Rather than messing with how lein orders its dependencies, we can work around this by adding a mime.types to qu's own resources directory.  The file in this PR is a copy of the file in the ring.middleware.mime-extentions repo.

### CODE REVIEW

**Comment by marcesher** [comment]
Some more context: For the past few years, this has worked because ring.middleware.mime-extensions was ordered earlier than the aws-java-sdk in lein's classpath. As of 2.5, lein's classpath ordering appears to have changed, and aws-java-sdk is now ordered earlier. Because  ring-middleware.mime-extensions by default looks for mime.types on the entire classpath, it's now picking up the mime.types file in the aws-java-sdk jar before the one in ring.middleware.mime-extensions' own jar.

Basically, we've just been getting lucky for a few years. By putting the file in qu's resources directory, we guarantee that it'll get picked up first because lein orders project resources before dependencies. Yes, we're relying on that behavior not changing, but that seems like a reasonable bet. 

We can also talk to @cndreisbach about a suitable fix for ring.middleware.mime-extensions itself so that it does a targeted search for the mime.types file by default

**Comment by marcesher** [comment]
selfie-3
